nipype/pipeline/plugins/tests/test_callback.py

```diff
@@ -65,8 +69,10 @@ def test_callback_exception(tmpdir):
     assert so.statuses[1][1] == 'exception'
 
 def test_callback_multiproc_normal(tmpdir):
+    tmpdir.chdir()
+
     so = Status()
-    wf = pe.Workflow(name='test', base_dir=tmpdir.strpath)
+    wf = pe.Workflow(name='test')
```

Comment:
perhaps `base_dir` should be set to local wd. in this case the working directory will be in some other tmp directory